camel-core/src/test/java/org/apache/camel/component/file/FileConsumerExtendedAttributesTest.java

```diff
@@ -81,6 +82,9 @@ public void testBasicAttributesAsDefaultWithFilter() throws Exception {
 
     @Test
     public void testPosixAttributes() throws Exception {
+        if (FileUtil.isWindows()) {
```

Comment:
I was not able to use `Assume.assumeFalse`, because the test is not run with a junit4 runner. If I force junit4 (by using `@RunWith(JUnit4.class)`) `ContextTestSupport` can not do it's work.